tests/run

```diff
@@ -0,0 +1,2 @@
+#! /usr/bin/env bash
+python -m unittest discover --start-directory tests
```

Comment:
Agreed- sorry, that was left in from when I was doing the initial commits of this. Not sure if it's worth removing or just modifying in #14?